src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submissions.cljs

```diff
@@ -0,0 +1,119 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.submissions
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.paths :as paths]
+    [org.broadinstitute.firecloud-ui.utils :as utils]))
+
+(defn submission-running? [submission]
+  (let [status (get submission :status)]
```

Comment:
It only works for keywords.  If the key is a string, you can still do `(map "key")`